code/src/components/SubmitButton.css

```diff
@@ -0,0 +1,28 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap');
+
+
+* {
+  font-family: 'Roboto Mono', monospace;
+  text-rendering: optimizeLegibility;
+}
+
+.submit-button {
+    border-radius: 20px;
+    padding: 5px;
+    background-color: #ffadad;
+    border: 1px solid #ffadad; 
+    cursor: pointer;
+    font-size: 14px;
+    display: flex;
+    align-items: center;
+    margin-top: 7px;
+    
+}
+
+.submit-button:hover {
+  transform: scale(1.1);
+}
+
+.emoji {
+padding: 4px;
+}
```

Comment:
This makes your like-buttons oval. If you take away the padding on the emoji, you will get a circle